src/GovUk.Education.ExploreEducationStatistics.Data.Api/GovUk.Education.ExploreEducationStatistics.Data.Api.csproj

```diff
@@ -19,7 +19,6 @@
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.18.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.2" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.2.3" />
```

Comment:
Interesting, assume we've never used blob storage directly in the data API project?

Reply:
We do, for example storing and retrieving Permalinks, but that code is using the newer v12 SDK `Azure.Storage.Blobs` which is a dependency of `ExploreEducationStatistics.Common` that Data.Api depends on.

I've removed all references to the deprecated `Microsoft.Azure.Storage.Blob` but don't totally get rid of it just yet.

Our `Common.Services.BlobStorageService` has a dependency on `TransferManager` which belongs 
 to `Microsoft.Azure.Storage.DataMovement`. But that in turn depends on the deprecated `Microsoft.Azure.Storage.Blob`.